node/dgram.ts

```diff
@@ -455,12 +455,12 @@ export class Socket extends EventEmitter {
         return;
       }
 
-      let flags = 0;
+      let flags: number | undefined = 0;
 
       if (state.reuseAddr) {
         flags |= UV_UDP_REUSEADDR;
       }
-      if (state.ipv6Only) {
+      if (state.ipv6Only && UV_UDP_IPV6ONLY) {
```

Comment:
Bitwise assigned won't change the variable if the assignment is undefined, however TypeScript complains

Reply:
I guess we should do `UV_UDP_IPV6ONLY!` below if this is only TypeScript issue